README

```diff
@@ -193,6 +193,13 @@ REQUIREMENTS:
         under all circumstances. In fact, systemd-hostnamed will warn
         if nss-myhostname is not installed.
 
+        nss-systemd must be enabled on systemd systems, as that's required for
+        DynamicUser= to work. Note that we ship services out-of-the-box that
+        make use of it now, hence nss-systemd is not an option.
```

Comment:
“nss-systemd is not an option” sounds like it can’t be used, which is the opposite of the intended meaning. Perhaps “nss-systemd is not optional” or “running without nss-systemd is not an option”?